assets/base.css

```diff
@@ -1428,6 +1428,10 @@ button.shopify-payment-button__button--unbranded {
   display: flex;
 }
```

Comment:
- [ ] Design question: i feel like the add to cart could be using a bit more space to not be so close to the price. [screenshot](https://screenshot.click/17-31-3aedw-6tso7.png)

Reply:
Thanks Ludo! Let's park product card improvements for now, we can revisit once we start working on product cards.